### ISSUE TITLE
the type is QUESTION, is not coded. Please make a call

### ISSUE THREAD

**Issue by unity-emilk** [uncrustify/uncrustify#4023]
Using `Uncrustify-0.77.1_f` on Mac M1, from Homebrew.

I tried to generate a debug file, but following the suggested `uncrustify -p debug.txt -c pathToUsedConfig.cfg -f toBeFormatedFile.cpp` did not generate a file.

Error:
```
ParsingFrame::pop (indent_text:1351): orig line is   19, orig col is   28, type is     QUESTION,
ParsingFrame::pop (indent_text:1351): the type is QUESTION, is not coded. Please make a call.
```

on file:
```
class foo {
public:
    bool var;
 
    foo(bool x) { var = x; }
 
    bool get()
    {
        return var;
    }
};


int main() {
    foo f(false);

    const int b = true ? 2 : 3;

    const int a = !f.get() ? 2 : 3;
}
```

I believe its the use of a ternary operator combined with a method call that causes the error.

**Comment by guy-maurel** [comment]
This is a bug.
We shall repair it, thanks for the call.

**Comment by micheleCTDE** [comment]
I tried to replicate the behavior using current master version, using the source code above and default configuration. But it does not seem to cause the same effect.
Could you upload a config file that can be used to recreate the problem?

**Comment by unity-emilk** [comment]
It seems that
```
indent_member_single=true
```

will cause the error

**Comment by micheleCTDE** [comment]
Great, thank you.
I can reproduce the issue. Will push a fix later tonight

**Comment by micheleCTDE** [comment]
I just merged PR #4025, which is a fix for this problem.
@unity-emilk please test with latest code and if it solves the issue for you too, please close this issue report.

**Comment by unity-emilk** [comment]
Works, thanks!
